### DISCUSSION TITLE
GitHub Pages: First Scenario: ue-app1 is referenced but never created

### DISCUSSION THREAD

**Discussion by reckseba** [InterDigitalInc/AdvantEDGE#434] [Q&A] [answered]
Hey guys,

pretty interesting and exciting piece of tech you have crafted - thanks for that!

One thing: I'm constantly moving through the documentation and found that the following is a little odd:

[Line 76](https://github.com/InterDigitalInc/AdvantEDGE/blob/gh-pages/docs/usage/usage-first-scenario.md?plain=1#LL76) says "iperf-cloud-client" which gets copied over as "ue-app2" and gets [later](https://github.com/InterDigitalInc/AdvantEDGE/blob/gh-pages/docs/usage/usage-first-scenario.md?plain=1#L96) referenced as "ue-app1" (I guess). Or did I get it wrong somehow?

And while we're here: Why isn't ue-app2 not connecting to iperf-fog-server and instead connecting to iperf-cloud-server like ue-app1 (iperf-cloud-client)? Story wise would have made sense to me if one ue checks out the cloud and the other the fog to show off the :exploding_head: difference. Blown me away anyway. But others might stumble upon that, too maybe.

Thanks
Sebastian

**Comment by dilallkx** [comment, marked as answer]
Hi @reckseba,

Thank you for the feedback!

WRT the documentation, this first scenario example shows the user some of the basic controls to create a new scenario (e.g. network element creation & cloning). The cloning of _ue1_ creates a second UE, both of which keep the _ue1_ configuration (e.g. both start an iperf client that communicates with the cloud server instance). I agree it would make more sense to have UE2 connect to the fog server instead, however we wanted to show the user how to use the dashboards to observe the run-time network characteristics and wanted to show how multiple data flows to a single server (iperf-cloud-server) could be observed in the AdvantEDGE frontend.

Demo1, however, does exactly what you suggest; the (basic) edge selection algorithm picks the closest available edge service instance that a UE app should connect to. As the UE changes PoA (e.g., via manually injected UE mobility events), the algorithm recalculates the closest edge node and triggers a reconnection to the closest edge service. Give it a try :)

Finally, for the question about the documentation stating to select ue-app1, this seems correct to me. We can observe the ue1 network characteristics. Of course, the same could be done for ue2, but the example is for ue1. The example also says to then check the cloud application iperf-cloud-server to observe traffic from both ue1 and ue2. I hope this clarifies the example!

Again, thank you for your feedback! Please let us know if you have any other questions...
Regards!
